dlp/api/DlpSample/RiskAnalysis.cs

```diff
@@ -74,14 +75,14 @@ class RiskAnalysis
             var submittedJob = dlp.CreateDlpJob(
                 new CreateDlpJobRequest
                 {
-                    ParentAsProjectName = new Google.Cloud.Dlp.V2.ProjectName(callingProjectId),
+                    ParentAsProjectName = new ProjectName(callingProjectId),
                     RiskJob = config
                 });
 
             // Listen to pub/sub for the job
             var subscriptionName = new SubscriptionName(callingProjectId, subscriptionId);
-            SubscriberClient subscriber = SubscriberClient.Create(
-                subscriptionName, new[] { SubscriberServiceApiClient.Create() });
+            SubscriberClient subscriber = SubscriberClient.CreateAsync(
```

Comment:
Did they change the API? Why did you change the original code? It's better to call sync method directly than call async method and then .Result

Reply:
Yes, they changed the API.  I know it's better to await the result, but that would require making all, or most of the functions in the call stack async, which is overkill for this sample that otherwise does nothing asynchronous.